app/views/notes/_card.html.erb

```diff
@@ -52,16 +52,16 @@
       <a class="ellipsis bottom-right" data-toggle="dropdown"><i class="fa fa-ellipsis-h" style="color : #666; font-size:15px; float:right; cursor:pointer;"></i></a>
 
       <ul class="dropdown-menu" style = "width: 150px; font-size:10px;">
-        <li><a>Total Views: </i> <%= number_with_delimiter(node.views) %> <span class="d-none d-lg-inline"><%= t('notes._notes.views') %></span></a></li>
+      <li><a>Total Views: </i> <%= number_with_delimiter(node.views) %> <span class="d-none d-lg-inline"><%= translation('notes._notes.views') %></span></a></li>
         <li><a>Total Likes: <%= node.cached_likes %></a></li>
         <div class="content" style="width: 100%" >
           <% if @compact.nil? %>
 
             <li class="buttons mb-2"><% if @current_user && @current_user.can_moderate? %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %></li>
```

Comment:
Hi @stephaniequintana is this the line where you weren't sure? It should be OK to change the `t()` function here. The code is very similar, and has the same function, and shouldn't interfere with your changes. 

If you like you can accept this suggestion:

```suggestion
            <li class="buttons mb-2"><% if @current_user && @current_user.can_moderate? %><a style="width: 90px" class="btn btn-outline-secondary" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= translate('notes._notes.spam') %></a><% end %></li>
```

Thank you!!

Reply:
Yes, that was the function in question. Thank you!